labelme/app.py

```diff
@@ -36,7 +36,8 @@
 from labelme.widgets import LabelQListWidget
 from labelme.widgets import ToolBar
 from labelme.widgets import ZoomWidget
-
+from skimage import io as skio
```

Comment:
`scikit-image` is not included in the dependencies. Isn't that possible to use PIL to open mim file?

Reply:
PIL can not open the mim file. If skimge can not be added to the dependency, this pull request will be closed.

Reply:
It can be an optional dependency.